docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -68,6 +68,11 @@ If you configure Ansible to ignore undefined variables, you may want to define s
 
 The variable value will be used as is, but the template evaluation will raise an error if it is undefined.
 
+A convenient way of requiring a variable to be overridden is to give it an undefined value using the ``Undef`` keyword. This can be useful in a role's defaults::
```

Comment:
Please use a code-block here instead of :: .  The translation of Ansible docs can ignore content within a code-block but has problems with ::.  See https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#syntax-highlighting-pygments for an example of a code-block.

Reply:
I had copypasted from another section in that file. Is it a problem that many of the other segments don't use a code-block?

Reply:
It makes things harder for translation. If you feel like changing the rest, sure, but don't feel you have to.